sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/FileFormatWriter.scala

```diff
@@ -183,15 +183,16 @@ object FileFormatWriter extends Logging {
       val commitMsgs = ret.map(_.commitMsg)
 
       committer.commitJob(job, commitMsgs)
-      logInfo(s"Job ${job.getJobID} committed.")
+      logInfo(s"Job ${job.getConfiguration.get("mapreduce.job.id")} committed.")
```

Comment:
we should log something here, but `mapreduce.job.id` is not useful, how about `description.uuid`?

Reply:
SparkHadoopWriter needs a similar change, then, BTW